addons/calendar/models/calendar_event.py

```diff
@@ -548,6 +570,20 @@ def unlink(self):
         self.env['calendar.alarm_manager']._notify_next_alarm(partner_ids)
         return result
 
+    def copy(self, default=None):
+        """When an event is copied, the attendees should be recreated to avoid sharing the same attendee records
+         between copies
+         """
+        self.ensure_one()
+        if not default:
+            default = dict()
+        # We need to make sure that the attendee_ids are recreated with new ids to avoid sharing attendees between events
+        # The copy should not have the same attendee status than the original event
+        default.update({'partner_ids': [Command.set([])], 'attendee_ids': [Command.set([])]})
```

Comment:
You don't have to pass a literal dict to `dict.update`, you can just give it kwargs.